tests/runtests.py

```diff
@@ -171,6 +177,22 @@ def setup_additional_options(self):
             action='store_true',
             help='Do not run any tests. Simply start the daemons.'
         )
+        self.add_option(
+            '--custom',
+            default=False,
+            action='store_true',
+            help='Addtional directories to scan for tests'
+        )
+        self.add_option(
+            '--customdir',
+            default='',
+            help='Directory to scan for additional tests. (Requires that --custom also be set)'
```

Comment:
Is the `--custom` flag really required? At least from the updated doc page, it reads the other way around... `--custom` requires `--custom-dir`, but if only `--custom-dir` is passed then it is used as an extra test directory in addition to the regular salt tests.